cleaned_data/SpanishLanguageRecordings/rfta:171.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">20220119_Duarte_Mitsy-Gissela</identifier>
+   <identifier type="pid">rfta:171</identifier>
```

Comment:
We haven't ingested this recording yet, so you can leave off identifier[@type="pid"] altogether.